.solhint.json

```diff
@@ -10,14 +10,12 @@
       , "func-param-name-mixedcase": "error"
       , "func-visibility": ["error", { "ignoreConstructors": true }]
       , "imports-on-top": "error"
-      , "max-line-length": ["error", 100]
       , "modifier-name-mixedcase": "error"
       , "named-parameters-mapping": "error"
       , "no-console": "error"
       , "no-empty-blocks": "error"
       , "no-unused-vars": "error"
       , "no-global-import": "off"
-      , "private-vars-leading-underscore": ["error", { "strict": true }]
```

Comment:
@alxiong I thought the issue with this was fixed by updating solhint.

Reply:
the main problem is conflict requirements by forge linters and solhint linters.
when enabled, I would have:
```
contracts/src/libraries/PlonkVerifier.sol
   28:5  error  'COSET_K1' should start with _                           private-vars-leading-underscore
   30:5  error  'COSET_K2' should start with _                           private-vars-leading-underscore
   32:5  error  'COSET_K3' should start with _                           private-vars-leading-underscore
   34:5  error  'COSET_K4' should start with _                           private-vars-leading-underscore
```

but naming them `_COSET_K4` would upset `forge build`